Runtime/BacktraceDatabase.cs

```diff
@@ -261,10 +264,39 @@ private void Start()
             {
                 return;
             }
+            string breadcrumbPath = string.Empty;
+            string breadcrumbArchive = string.Empty;
+
+            if (Breadcrumbs != null)
+            {
+                breadcrumbPath = Breadcrumbs.GetBreadcrumbLogPath();
+                breadcrumbArchive = Breadcrumbs.Archive();
+            }
             // load reports from hard drive
-            LoadReports();
+            LoadReports(breadcrumbPath, breadcrumbArchive);
             // remove orphaned files
             RemoveOrphaned();
+
+            // send minidump files generated by unity engine or unity game, not captured by Windows native integration
+            // this integration should start before native integration and before breadcrumbs integration
+            // to allow algorithm to send breadcrumbs file - if the breadcrumb file is available
+#if UNITY_STANDALONE_WIN
+
+            var isEnabled = Enable && Configuration.SendUnhandledGameCrashesOnGameStartup && isActiveAndEnabled;
+            var hasNativeConfiguration = _client && _client.NativeClient != null && _client.NativeClient is IStartupMinidumpSender;
+            if (isEnabled && hasNativeConfiguration)
+            {
+                var client = _client.NativeClient as IStartupMinidumpSender;
+                var attachments = _client.GetNativeAttachments();
+                if(string.IsNullOrEmpty(breadcrumbArchive))
+                {
+                    attachments.Add(breadcrumbArchive);
+                }
```

Comment:
is this valid? Do you want to add `breadcrumbArchive` only if it is `null` or empty?

Reply:
I messed this up before finalizing pull request. Now it should work. Thank you!